.github/workflows/claude.yml

```
@@ -39,9 +34,31 @@ jobs:
-          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
-          model: "claude-sonnet-4-20250514"
-          allowed_tools: >-
-            Bash(git:*),Bash(gh:*),Bash(pnpm:*),Bash(npm:*),Bash(npx:*),
-            Bash(mkdir:*),Bash(mv:*),Bash(cp:*),Bash(touch:*),Bash(ls:*),
-            Bash(cat:*),Bash(echo:*),Bash(pwd:*),Bash(open:*)
+          claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
+
+          # This is an optional setting that allows Claude to read CI results on PRs
+          additional_permissions: |
+            actions: read
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)